gax/src/main/java/com/google/api/gax/rpc/ApiCallContext.java

```diff
@@ -56,16 +56,22 @@
   /**
    * Returns a new ApiCallContext with the given timeout set.
    *
-   * <p>This timeout only applies to a single RPC call; if timeouts are configured, the overall time
-   * taken will be much higher.
+   * <p>This sets the maximum amount of time a single unary RPC attempt can take. If retries are
```

Comment:
Please also clarify the scope of ApiCallContext in `ApiCallContext` class level documentation itself (that it represents an actual "unit of work", but also exposes single rpc-specific stuff, like the `withTimeout()` thing). it should be clear what "api call" means here and that it may imply 1+ actual rpcCalls, and that `ApiCallContext` is basically a conglomeration of random state-like objects which accompany the actual "unit of work".